spec/redis_connection_spec.rb

```diff
@@ -52,4 +52,47 @@ def self.name
     model_class.redis = Redis.new
     expect(model_class.redis).to_not eq Redcord::Base.redis
   end
+
+end
+
+describe Redcord do
+  let!(:model_class0) do
```

Comment:
lets try to not define runtime classes this way unless we have to.

Suggestion:
```ruby
class ModelClass0 < T::Struct
...
end
```